projects/ccf-shared/src/lib/services/data-source/api-endpoint.service.ts

```diff
@@ -131,6 +131,51 @@ export class ApiEndpointDataSourceService implements DataSource {
     );
   }
 
+  // TODO: Remove mock code when this.api.biomarkerTreeModel is available
+  formBiomarkerNode(id,parent,children,nodeType=''): OntologyTreeNode {
+    return {
+      ['@id']: id,
+      id: id,
+      label: id,
+      parent: parent??'',
+      children: children??[],
+      synonymLabels: [],
+      ['@type']: 'OntologyTreeNode',
+      nodeType: nodeType
+    };
+  }
+
+  /**
+   * Get the cell type tree model.
+   *
+   * @returns An observable emitting the results.
+   */
+  @Cacheable(CACHE_CONFIG_NO_PARAMS)
+  getBiomarkersTreeModel(): Observable<OntologyTreeModel> {
+    const a= {
+      root:'biomarkers',
+      nodes: {
+        'biomarker1' : this.formBiomarkerNode('biomarker1','biomarkers',[],'gene'),
+        'biomarker2' : this.formBiomarkerNode('biomarker2','biomarkers',[],'gene'),
+        'biomarker3' : this.formBiomarkerNode('biomarker3','biomarkers',[],'protein'),
+        'biomarker4' : this.formBiomarkerNode('biomarker4','biomarkers',[],'protein'),
+        'biomarker5' : this.formBiomarkerNode('biomarker5','biomarkers',[],'lipid'),
+        'biomarker6' : this.formBiomarkerNode('biomarker6','biomarkers',[],'lipid'),
+        'biomarkers' : this.formBiomarkerNode('biomarkers','',['biomarker1','biomarker2','biomarker3','biomarker4','biomarker5','biomarker6']),
+      }
+    };
+    return of(a as OntologyTreeModel);
+  }
+
+  // TODO: Use this code when this.api.biomarkerTreeModel is available
```

Comment:
@bherr2 Is this not available yet?

Reply:
Correct. I'll add the necessary code but where they put the mock data is good.